recipe/meta.yaml

```diff
@@ -6,9 +6,14 @@ package:
   version: {{ version }}
 
 source:
+  # Project URL:
+  #
   # url: http://geuz.org/{{ name }}/src/{{ name }}-{{ version }}.tgz
   # sha256: 8d1c00c1018f96b4b97655482e57dcb0ce42ae2f1d349cd6d4191e7848d9ffe9
-  url: http://deb.debian.org/debian/pool/main/g/gl2ps/gl2ps_1.4.2+dfsg1.orig.tar.xz
+  #
+  # Download from the Debian CND because connections to the above project URL
```

Comment:
hmm, is it really just on some builders?  if so, I would suspect it has something to do with IT's access restrictions from builder-network